packages/dds/tree/src/test/shared-tree/summaryBenchmarkSizes.ts

```diff
@@ -0,0 +1,173 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "assert";
+import { IsoBuffer } from "@fluidframework/common-utils";
+import { makeRandom } from "@fluid-internal/stochastic-test-utils";
+import { TransactionResult } from "../../checkout";
+import { FieldKinds, singleTextCursor } from "../../feature-libraries";
+import { ISharedTree } from "../../shared-tree";
+import { FieldKey, rootFieldKey, rootFieldKeySymbol, TreeValue } from "../../tree";
+import { brand } from "../../util";
+import { TestTreeProvider } from "../utils";
+import { fieldSchema, GlobalFieldKey, namedTreeSchema, SchemaData } from "../../schema-stored";
+// eslint-disable-next-line import/no-internal-modules
+import { PlacePath } from "../../feature-libraries/sequence-change-family";
+
+const globalFieldKey: GlobalFieldKey = brand("globalFieldKey");
+
+describe("Summary size benchmark", () => {
```

Comment:
the idea with our benchmarks is that they output performance information which we can track over time.

These tests would perform no differently if we optimized summary size to be 80% smaller, so I'm not sure they really are benchmarks.

We should probably have a todo to hook this up to the benchmark data collection / reporting.
```suggestion
// TODO: report these sizes as benchmark output which can be tracked over time.
describe("Summary size benchmark", () => {
```

Reply:
added TODO comment